### PR TITLE
Fixes #5006 User Default Billing Facility

### LINKED ISSUE #5006: Add Default Billing Facility for Providers
We support in the database for a user/provider to have an assigned default billing facility.  In the user admin add / edit functionality we need to be able to set the default billing facility for a provider.  This supports relationships needed by CCDA as well as for the AMC calculations.

### PR DESCRIPTION
**Pull request by adunsulag**
We track it in the database but there wasn't a mechanism for setting a
provider's default billing facility. Since we need that for AMC and CCDA
I've gone ahead and added that in.

### CODE REVIEW

**Comment by stephenwaite** [comment]
nice foresight from @sjpadgett adding that to users, https://github.com/openemr/openemr/pull/4661 !

nice

**Review by stephenwaite — APPROVED** [review]

**Comment by sjpadgett** [comment]
I don't know why I didn't go ahead and add to user UI. Maybe to give @adunsulag something to do!:)

**Review by bradymiller — APPROVED** [review]

File: interface/usergroup/user_admin.php [hunk 1]
```diff
@@ -577,7 +577,7 @@ class="form-control" value="<?php echo attr($frow['id']); ?>"><?php echo text($f
 
   </tr>
     <tr>
-        <td><span class=text><?php echo xlt('Default Billing Facility'); ?>: </span></td><td><select name=billing_facility_id style="width:150px;" class="form-control">
+        <td><span class=text><?php echo xlt('Default Billing Facility'); ?>: </span></td><td><select name="billing_facility_id" style="width:150px;" class="form-control">
```
Comment (stephenwaite) [inline interface/usergroup/user_admin.php:580]: missed eagle scout points, dang it

Comment (adunsulag) [reply interface/usergroup/user_admin.php:580]: Not sure I'm following.  Do I need to fix something here?

Comment (stephenwaite) [reply interface/usergroup/user_admin.php:580]: nope, just joking about approving it prematurely :)

**Pull request merged**